packages/terra-switch/src/terra-dev-site/doc/switch/UpgradeGuide.2.doc.mdx

```diff
@@ -0,0 +1,6 @@
+import { Badge } from 'terra-switch/package.json?dev-site-package';
```

Comment:
This file is not needed for the first release.

Reply:
Removed